cli/formatting.go

```diff
@@ -0,0 +1,192 @@
+			prettifyAsset(delegatedNet),
+		),
+		fmt.Sprintf("     used:|%s", prettifyBytes(int64(account.NetLimit.Used))),
+		fmt.Sprintf("     available:|%s", prettifyBytes(int64(account.NetLimit.Available))),
+		fmt.Sprintf("     limit:|%s", prettifyBytes(int64(account.NetLimit.Max))),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatCPUBandwidth(account *eos.AccountResp, config *columnize.Config) string {
+	delegatedCPU := account.TotalResources.CPUWeight.Sub(account.SelfDelegatedBandwidth.CPUWeight)
+
+	output := []string{
+		"cpu bandwidth:",
+		fmt.Sprintf("     staked:|%s|(total stake delegated from account to self)",
+			prettifyAsset(account.SelfDelegatedBandwidth.CPUWeight),
+		),
+		fmt.Sprintf("     delegated:|%s|(total stake delegated from account from others)",
+			prettifyAsset(delegatedCPU),
+		),
+		fmt.Sprintf("     used:|%s", prettifyTime(int64(account.CPULimit.Used))),
+		fmt.Sprintf("     available:|%s", prettifyTime(int64(account.CPULimit.Available))),
+		fmt.Sprintf("     limit:|%s", prettifyTime(int64(account.CPULimit.Max))),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatBalances(account *eos.AccountResp, config *columnize.Config) string {
+	totalStaked := account.SelfDelegatedBandwidth.NetWeight.Add(account.SelfDelegatedBandwidth.CPUWeight)
+	totalUnstaking := eos.Asset{
+		Amount: 0,
+		Symbol: account.CoreLiquidBalance.Symbol,
+	}
+	if account.RefundRequest != nil {
+		totalUnstaking = account.RefundRequest.CPUAmount.Add(account.RefundRequest.NetAmount)
+	}
+	total := totalUnstaking.Add(totalStaked).Add(account.CoreLiquidBalance)
+
+	output := []string{
+		fmt.Sprintf("%s balances:", account.CoreLiquidBalance.Symbol.Symbol),
+		fmt.Sprintf("     liquid:|%s", prettifyAsset(account.CoreLiquidBalance)),
+		fmt.Sprintf("     staked:|%s", prettifyAsset(totalStaked)),
+		fmt.Sprintf("     unstaking:|%s", prettifyAsset(totalUnstaking)),
+		fmt.Sprintf("     total:|%s", prettifyAsset(total)),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatProducers(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"producers:",
+		fmt.Sprintf("     %s", prettifyAccounts(account.VoterInfo.Producers)),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func prettifyBytes(b int64) string {
+	const u = 1024
+	if b < u {
+		return rightAlignColumnize(fmt.Sprintf("%d", b), "bytes")
+	}
+	div, exp := int64(u), 0
+	for n := b / u; n >= u; n /= u {
+		div *= u
+		exp++
+	}
+	value := float64(b) / float64(div)
+	unit := fmt.Sprintf("%ciB", "KMGTPE"[exp])
+
+	precision := 3
+	if value >= 100 {
+		precision = 1
+	} else if value >= 10 {
+		precision = 2
+	}
+
+	return rightAlignColumnize(strconv.FormatFloat(value, 'f', precision, 64), unit)
+}
+
+func prettifyTime(micro int64) string {
+	value := float64(micro)
+	unit := "μs"
+	if value > 1000000*60*60 {
+		value /= float64(1000000 * 60 * 60)
+		unit = "hr"

```

Comment:
`h` seems more common

Reply:
I chose to use the cleos unit per the original issue discussion, completely agree if we want to diverge from their units.  @abourget had originally suggested matching their units